conf/evolutions/default/53.sql

```diff
@@ -0,0 +1,15 @@
+# --- !Ups
+ALTER TABLE validation_task_interaction
+  ADD is_mobile INT;
+
+ALTER TABLE label_validation
+  ADD is_mobile INT;
+
```

Comment:
Is there a reason why you made both these columns nullable? I don't see why we can't make them `NOT NULL`, which would be ideal. We know that all the past interactions have not been mobile, so we can backfill the rest of the table with `is_mobile=FALSE`.